components/buttons/buttons.css

```diff
@@ -0,0 +1,43 @@
+.apex-btn{
+    color: var(--apex-text-btn);
+    padding: var(--apex-btn-padding);
+    border: none;
+    border-radius: var(--apex-btn-border-radius);
+    cursor: pointer;
+}
+.apex-primary-btn{
+    background-color: var(--primary-btn);
+}
+.apex-secondary-btn{
+    background-color: var(--secondary-btn);
+}
+.apex-link{
+    color: var(--apex-text-link-color);
+    text-decoration: none;
+}
+.apex-link-btn{
+    background-color: var(--link);
+}
+.apex-cart-btn{
+    background-color: var(--cart-btn);
+}
+.apex-like-btn{
```

Comment:
Just give some margin right between icon and text inside button.

Reply:
Yes I 'll do that ,that would make it look good